lib/features/copy-paste/index.js

```diff
@@ -1,17 +1,18 @@
 import CopyPasteModule from 'diagram-js/lib/features/copy-paste';
 
-import ChoreoCopyPaste from './ChoreoCopyPaste';
+import BpmnCopyPaste from 'bpmn-js/lib/features/copy-paste/BpmnCopyPaste';
+import ChoreoModdleCopy from './ChoreoModdleCopy';
 
-/**
- * We completely overwrite the standard bpmnCopyPaste and register ChoreoCopyPaste as its replacement. We needt to do this,
- * becuase bpmnCopyPaste registers a function in its constructor
- * EventBus.on('element.paste') that we do not want to execute an cannot otherwise 'nicely' overwrite and prevent from
- * bpmnCopyPaste from executing on paste.
- */
 export default {
   __depends__: [
     CopyPasteModule
   ],
-  __init__: ['bpmnCopyPaste'],
-  bpmnCopyPaste: ['type', ChoreoCopyPaste]
+  __init__: [
+    'moddleCopy',
+    'bpmnCopyPaste'
+  ],
+  moddleCopy: ['type', ChoreoModdleCopy],
+  bpmnCopyPaste: ['type', BpmnCopyPaste]
+  // We can use the original BpmnCopyPaste, as we add a low prio event listener in ChoreoModdleCopy that takes care
+  // of the chore specific things we need to handle.
```

Comment:
Do we have to add `bpmnCopyPaste` ourselves here, then? If it is added by `bpmn-js` anyways?

Reply:
No we don't ;). I will remove it